src/Microsoft.Sbom.Api/Workflows/Helpers/ExternalDocumentReferenceGenerator.cs

```diff
@@ -22,49 +22,43 @@ public class ExternalDocumentReferenceGenerator : IJsonArrayGenerator<ExternalDo
 {
     private readonly ILogger log;
 
-    private readonly ISbomConfigProvider sbomConfigs;
-
     private readonly IEnumerable<ISourcesProvider> sourcesProviders;
 
     private readonly IRecorder recorder;
 
+    public ISbomConfig SbomConfig { get; set; }
```

Comment:
Can `SbomConfig` and `SpdxManifestVersion` be set more than once? Not for this PR, but a small follow-up PR could easily add a class that makes these write-once

Reply:
No they should only be write-once, I will create a work item to make sure this change gets taken up in a separate PR.